src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensionRoutes.scala

```diff
@@ -60,7 +61,9 @@ trait GoogleExtensionRoutes extends ExtensionRoutes with UserInfoDirectives with
             pathPrefix("key") {
               get {
                 complete {
-                  googleExtensions.getPetServiceAccountKey(WorkbenchUser(userInfo.userId, userInfo.userEmail), GoogleProject(project)).map(x => StatusCodes.OK -> x)
+                  import spray.json._
+                  // parse json to ensure it is json and tells akka http the right content-type
+                  googleExtensions.getPetServiceAccountKey(WorkbenchUser(userInfo.userId, userInfo.userEmail), GoogleProject(project)).map { x => StatusCodes.OK -> x.parseJson }
```

Comment:
i just realized this, but the other endpoint for getting pet SA keys needs to be parsed into json

Reply:
+1